templates/vue/repo-search.tmpl

```diff
@@ -0,0 +1,81 @@
+<script type="text/x-template" id="repo-search-template">
```

Comment:
I don't think the file should be in `templates/vue` dir. We classify the templates according features not the front-end framework it used.